draft-demarco-oauth-nonce-endpoint.md

```diff
@@ -41,13 +45,19 @@ informative:
   BCP195: BCP195
 
 --- abstract
-This document defines the Nonce Endpoint for OAuth 2.0 implementations [RFC6749]. It details how an Authorization Server generates and issues opaque Nonces and how a client can learn about this endpoint to obtain a Nonce generated an Authorization Server or another entity.
+This document details how a client can learn about an endpoint to obtain a Nonce generated by a server (Nonce endpoint) and provides security guidelines for server-provided Nonce generation and verification.
 
---- middle
+--- middle 
 
 # Introduction
 
-This specification presents a comprehensive guide to the Nonce endpoint in OAuth 2.0 implementations [RFC6749]. It describes in detail how a client can request and receive a server-generated Nonce, which is a unique, one-time use, opaque string. This document provides in-depth insights into the cryptographic methods used in generating Nonces to protect the confidentiality of the information associated with them. In addition, it is a resource for developers and system architects who desire to strengthen the scalability, security, and efficiency of their systems while using OAuth 2.0.
+In different scenarios, it is required for a client application (e.g.,  an OAuth 2.0 Client) to present a Proof of Possession (PoP) of a private key in its control to a server (e.g., an OAuth 2.0 Authorization Server).
```

Comment:
"in different scenarios" is not the way I want to start the introduction.

I suggest to focus this PR only on the additional features/fixes/considerations reducing the editorial impacts on the well established text, separate issue for the editorials will be welcome

Reply:
I got it, I will close this PR and work on specific issues/PRs.